ansys/mapdl/reader/common.py

```diff
@@ -107,9 +107,9 @@ def read_record(self, pointer, return_bufsize=False):
 
         """
 
-        if self._cfile is not None:
-            record = self._cfile.read_record(pointer, return_bufsize)
-            return record
+        # if self._cfile is not None:
+        #     record = self._cfile.read_record(pointer, return_bufsize)
+        #     return record
 
```

Comment:
Why would we need to remove this part? And also, if it needs to be removed, we might be better off just deleting it rather than leaving it as dead code.

Reply:
It's a good point, and actually I need to review this for stability since this seems to be causing an issue when files are hosted remotely. I'll come back with a fix for this.

Reply:
Awesome, no problem! Thanks @akaszynski!